pkg/deploy/strategy/recreate/recreate.go

```diff
@@ -88,6 +93,25 @@ func (s *RecreateDeploymentStrategy) Deploy(from *kapi.ReplicationController, to
 	return s.DeployWithAcceptor(from, to, desiredReplicas, nil)
 }
 
+// recordControllerWarnings records the replication controller warnings into a
+// deployment event sink.
+func (s *RecreateDeploymentStrategy) recordControllerWarnings(rc *kapi.ReplicationController) {
```

Comment:
@kargakis i can, but we are using sink everywhere else, also I don't think I have client for events in lifecycle (i can check).